.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+language: c
+cache:
+  directories:
+    - $HOME/.ghc
+    - $HOME/.cabal
+    - $HOME/.stack
+addons:
+  apt:
+    packages:
+      - libgmp-dev
+before_install:
+  - mkdir -p ~/.local/bin
+  - export PATH=$HOME/.local/bin:$PATH
+  - curl -L https://www.stackage.org/stack/linux-x86_64 | tar xz --wildcards --strip-components=1 -C ~/.local/bin '*/stack'
+install:
+  - echo "$(ghc --version) [$(ghc --print-project-git-commit-id 2> /dev/null || echo '?')]"
```

Comment:
The `ghc` on the PATH is likely to be different from Stack's. Since the GHC version is specified by stack.yaml I think it's not too important to print the version we are using. If we do want to do this though, I think we should find out how to ask Stack what GHC version it's going to use based on what's in stack.yaml instead.